frontend/src/utils/results.js

```diff
@@ -109,8 +109,19 @@ function makeTableHeaders(message, colorMap) {
         }
         return knowledge_graph.nodes[value[0].id].name || value[0].id;
       },
+      disableSortBy: true,
     };
   });
+  if (results[0].score) {
+    const scoreColumn = {
+      Header: 'Score',
+      id: 'score',
+      accessor: (row) => Math.round(row.score * 1000) / 1000,
```

Comment:
What assumption(s) is this making about the range of the scores?

Reply:
This will round scores to the 1/1000th. I was getting scores from Robokop that had quite the number of decimal places. It didn't look very good. But we can do it smarter if needed. I'm assuming scores can vary between ARAs, some being between 0-1, others being well over 1?

Reply:
If the goal is to limit the display length, we might want to put a limit on total digits instead somehow, but this will work for now.